src/arrays/referencing_array_and_its_elements/medium/q9/EveryEven.java

```diff
@@ -0,0 +1,37 @@
+package arrays.referencing_array_and_its_elements.medium.q9;
+
+/*
+Make a Java program that initializes an integer array with 10 elements, whose values are randomly generated between 1
+and 1000 (inclusive of both ends), and print it. Then, create a second array of half of the size that copies the values
+from every even index element in the first array, and only finally print out the values of the second array to
+the console.
+
+Sample Output:
+    The random generated array:
```

Comment:
change "random" to "randomly" and carry that change through the rest of the program